### ISSUE TITLE
Image not working?

### ISSUE THREAD

**Issue by Disane87** [MRVDH/picnic-api#4]
HI, thank you for this great project! Currently I'm using your api for my project. Now I wanto to get the image of a product, but this isnt working:

```js
app.get('/image/:imageid.:extension', cors(corsOptions), async (req, res) => {
  const imageID = req.params.imageid;
  const extension = req.params.extension;
  const image = await picnicClient.getImage(imageID, ImageSizes.LARGE);
  // const base64Image = new Buffer.from(image.toString('binary'), 'base64')
  // res.writeHead(200, {
  //    'Content-Type': `image/${extension}`,
  //    'Content-Length': image.length
  //  });

  res.set('Content-Type', `image/${extension}`);
  res.set('Content-Length', image.length);
  res.set('Image-ID', `image/${extension}`);
 
  return res.end(image);
})

```

In postman I only get this:
![image](https://user-images.githubusercontent.com/956158/111883642-dbd5a200-89bc-11eb-9b4a-7204ae9affaa.png)

Am I doing it wrong?

**Comment by MRVDH** [comment]
Correct, the api sends a request with some incorrect headers. This needs to be fixed, which I will do soon, thanks for reporting. 👍 

If you want a temporary fix you can send a request like this: `'https://storefront-prod.nl.picnicinternational.com/static/images/IMAGE_ID/SIZE.png'`.
Example: `'https://storefront-prod.nl.picnicinternational.com/static/images/8560e1f1c2d2811dfefbbb2342ef0d95250533f2131416aca459bde35d73e901/medium.png'`
This is basically what the api does as well.

**Comment by Disane87** [comment]
Thank you for investigation! Actually I load them by the url manually like you proposed :)

**Comment by MRVDH** [comment]
Got distracted, but fixed it just now in v1.4.2. 👍
